docs/notebooks/langmuir_samples/_generate_noisy.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>Just thought I'd let you know how much nicer <code>pathlib</code>  is:</p><p> </p><p><code>import pathlib</code></p><p><code>samples_path = pathlib.Path("./") # not sure what you're going for with ./ here</code></p><p><code>path = samples_path / filename</code></p><p><code>#then in the next cell:</code></p><p><code>save_path = path.with_name(path.stem + "_noisy.npz")</code></p><p> </p><p><code><span class="ql-cursor">﻿</span></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Ahhh yeah, that is off.  I originally had this code in the `find_floating_potential` notebook and I didn't properly update everything when I copied it over.